src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.java

```diff
@@ -84,7 +84,7 @@ private void initComponents() {
         restultsJTable = new javax.swing.JTable();
 
         setDefaultCloseOperation(javax.swing.WindowConstants.DISPOSE_ON_CLOSE);
-        setTitle("Custom lookup");
```

Comment:
Did you mean to remove the title?

Reply:
Yes

Reply:
Erm... okay, but why?

Reply:
This String is already Set in [newSuggestionForm.form](https://github.com/BCDH/TEI-Completer/pull/40/files#diff-7baf7dda0096af18fe7d4f2f0ec5622f3880cdb1dbca28ca87b92955c4680625) And this is now duplicated.